vx_stuff.c

```diff
@@ -91,7 +91,7 @@ cvar_t		amf_part_teleport = {"gl_particle_telesplash", "0"};
 cvar_t		amf_part_fire = {"gl_particle_fire", "0"}; // 1
 cvar_t		amf_part_firecolor = {"gl_particle_firecolor", ""}; // 1
 cvar_t		amf_part_sparks = {"gl_particle_sparks", "0"}; // 1
-cvar_t		amf_part_traillen    = {"gl_particle_trail_lenght", "1"};
+cvar_t		amf_part_traillen    = {"gl_particle_trail_length", "1"};
```

Comment:
Could you make this (and the corresponding doc change) a separate commit too please. This will break configs as current configs have the misspelled cvar stored so it should be easy to spot in the git history. Thanks